prometheus/README.md

```diff
@@ -14,7 +14,9 @@ You can use this initialization action to create a new Dataproc cluster with Pro
     CLUSTER_NAME=<cluster_name>
     gcloud dataproc clusters create ${CLUSTER_NAME} \
         --region ${REGION} \
-        --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/prometheus/prometheus.sh
+        --num-masters 3 \
```

Comment:
Instead of using HA cluster you need to enable Zookeper explicitly for non-HA clusters: `--optioonal-components=ZOOKEEPER`